src/main.cpp

```diff
@@ -383,7 +384,7 @@ void testI2Cdevices(TwoWire *interface, uint8_t *device_list) {
   //reset device list
   memset(device_list,0,8);
 
-  while(supported_devices[devicenr] != 0x00) { //test until address is 0x00
+  while(supported_devices[devicenr] != 0x00 && !goingDormant) { //test until address is 0x00
```

Comment:
why is the goingDormant flag needed? did the I2C access interfere with sleep mode activation?  should we add a function instead to activate / suspend I2C access?